android/ui/src/main/java/com/trafi/ui/theme/Color.kt

```diff
@@ -45,22 +45,22 @@ fun MaasTheme.lightColors(
     onBackground = onBackground,
     onSurface = onSurface,
     onError = onError,
-    isLight = true
+    isLight = true,
```

Comment:
```suggestion
    isLight = true
```

Reply:
This was intentional 🙂 Kotlin 1.4 supports trailing parameter commas. The primary benefits are
- if we add a new parameter at the end, this line won't change in git
- easier to rearrange lines if you want to change the parameter order